.github/workflows/publish-latest.yml

```
@@ -87,3 +87,33 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+  log-failure-metric:
+    # Send data point 1 to metric PublishLatestFailure, if success
```

I think the comment is wrong?
```suggestion
    # Send data point 1 to metric PublishLatestFailure, if failure occurs
```